server/package.json

```diff
@@ -21,6 +21,7 @@
     "knex": "^0.21.5",
     "node-schedule": "^1.3.2",
     "node-telegram-bot-api": "^0.50.0",
+    "nodemon": "^2.0.4",
```

Comment:
O nodemon é uma ferramenta que não atualiza a aplicação sem precisar parar a aplicação e iniciar novamente correto? Se a utilização dele for por esse motivo, não há a necessidade em utiliza-lo. Pois atualmente a aplicação já é reiniciada.